index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html class="no-js">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title></title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="css/base.css" type="text/css" rel="stylesheet" />
+    <link href="css/layout.css" type="text/css" rel="stylesheet" />
+    <link href="css/module.css" type="text/css" rel="stylesheet" />
+    <link href="css/states.css" type="text/css" rel="stylesheet" />
+  </head>
+  <body class="section group">
+    <main>
+      <header class="container">
+        <h1>Andre's blog</h1>
+        <nav >
+          <a href="contact.html">Contact</a>
+          <a href="about.html">About</a>
+          <a href="projects.html">Projects</a>
+        </nav>
+      </header>
+      <article>
+        <h2>Lorem ipsum 1</h2>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque sagittis arcu eu ipsum aliquet suscipit. Nullam mollis elit at eros cursus malesuada. Mauris molestie metus sed quam tincidunt, a fringilla risus vehicula. Suspendisse potenti. Phasellus pulvinar pretium massa in accumsan. Vestibulum tristique ex blandit, hendrerit nisl ut, accumsan est. Morbi finibus urna viverra, ullamcorper nunc sed, scelerisque purus. Mauris posuere ullamcorper turpis sed aliquam.</p>
+      </article>
+      <section>
+        <article class="col span_12_of_12">
+          <h2>Lorem ipsum 2</h2>
+          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque sagittis arcu eu ipsum aliquet suscipit. Nullam mollis elit at eros cursus malesuada. Mauris molestie metus sed quam tincidunt, a fringilla risus vehicula. Suspendisse potenti. Phasellus pulvinar pretium massa in accumsan. Vestibulum tristique ex blandit, hendrerit nisl ut, accumsan est. Morbi finibus urna viverra, ullamcorper nunc sed, scelerisque purus. Mauris posuere ullamcorper turpis sed aliquam.</p>
+          <!-- <img src="front.jpg"> -->
+        </article>
+       <!--  <img src="front.jpg"> -->
+      </section>
+      <article class="col span_12_of_12">
+        <h2>Lorem ipsum 3</h2>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque sagittis arcu eu ipsum aliquet suscipit. Nullam mollis elit at eros cursus malesuada. Mauris molestie metus sed quam tincidunt, a fringilla risus vehicula. Suspendisse potenti. Phasellus pulvinar pretium massa in accumsan. Vestibulum tristique ex blandit, hendrerit nisl ut, accumsan est. Morbi finibus urna viverra, ullamcorper nunc sed, scelerisque purus. Mauris posuere ullamcorper turpis sed aliquam.</p>
+      </article>
+      <section class="col span_5_of_12">
+        <form>
+          <fieldset>
+            <legend>Questions?</legend>
+            <input id="given-name" name="given-name" type="text" placeholder="First name" required autofocus>
```

Comment:
It's usually a good idea to add a label to inputs in addition to the placeholder tag. This ensures a good UX when a user starts typing into a form field they don't forget what information they're being asked to give.